app/controllers/admin/words_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Admin::WordsController < ApplicationController
+  before_action
+
+  def new
+    @word = current_category.words.build
+  end
+
+  def create
+    @word = Word.new(word_params)
+    if @word.save
+      flash[:success]="Saved successfully!"
+      redirect_to admin_words_url
```

Comment:
Redirect back to the new word form:
`redirect_to new_admin_category_word_url(current_category)`